eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -1176,7 +1176,14 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
           // if we are fundee and we were waiting for them to send their first closing_signed, we don't have a lastLocalClosingFee, so we compute a firstClosingFee
           val lastLocalClosingFee = d.closingTxProposed.last.lastOption.map(_.localClosingSigned.feeSatoshis)
           val nextClosingFee = Closing.nextClosingFee(
-            localClosingFee = lastLocalClosingFee.getOrElse(Closing.firstClosingFee(d.commitments, d.localShutdown.scriptPubKey, d.remoteShutdown.scriptPubKey, nodeParams.onChainFeeConf.feeEstimator, nodeParams.onChainFeeConf.feeTargets)),
+            localClosingFee = lastLocalClosingFee.getOrElse {
```

Comment:
I think it would be clearer to completely skipe the `Closing.nextClosingFee` call in that case:

```scala
val nextClosingFee = if (d.commitments.localCommit.spec.toLocal == 0.msat) {
    // if we have nothing at stake there is no need to negotiate and we accept their fee right away
    remoteClosingFee
} else {
    Closing.nextClosingFee(
        localClosingFee = lastLocalClosingFee.getOrElse(Closing.firstClosingFee(d.commitments, d.localShutdown.scriptPubKey, d.remoteShutdown.scriptPubKey, nodeParams.onChainFeeConf.feeEstimator, nodeParams.onChainFeeConf.feeTargets)),
        remoteClosingFee = remoteClosingFee)
}
```

Reply:
Fixed in 772ffd9.